src/Rule/SubsetHandler.php

```diff
@@ -22,13 +22,8 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
         $result = new Result();
 
         if (!is_iterable($value)) {
-            $result->addError(
-                $rule->getIterableMessage(),
-                [
-                    'attribute' => $context->getAttribute(),
-                    'value' => (string) $value,
```

Comment:
`value` can be useful for custom messages.

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006452279

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006471638